mercury/tests/test_measurement.py

```diff
@@ -0,0 +1,75 @@
+import datetime
+
+import mock
+from django.test import TestCase
+from rest_framework.reverse import reverse
+
+from mercury.models import AGEvent
+
+
+def fake_event(event_uuid):
+    """
+        Mock a dummy AGEvent model
+    """
+    return AGEvent(
+        event_uuid=event_uuid,
+        event_name="fake event",
+        event_description="fake event",
+        event_date=datetime.datetime(2020, 2, 2, 20, 21, 22),
+        event_location="nyu",
```

Comment:
I think `mercury.models` is not going to be used soon, and we are transferring to `ag_data.models.AGEvent`, could you make a change to this?

Reply:
Yes, I can do it. Please give me some time.

Reply:
np

Reply:
I updated it and it passed all tests.